torchao/dtypes/aqt.py

```diff
@@ -282,6 +288,74 @@ def get_plain(self):
             f"Unpacking for tensor core tiled storage is not yet implemented"
         )
 
+@register_aqt_layout_cls("transposed")
+class TransposedAQTLayout(PlainAQTLayout):
```

Comment:
Why not use strides?

Reply:
main thing is that we need to transpose the weight before hand, while logically the weight still has the same dimensions, I think it's just a kind of packing. what do you mean by using strides? (doesn't transpose also just change the strides?)

Reply:
Yes, don't transpose and just use the strides

Reply:
Or put differently, when you implement `transpose` you change the sizes and strides so that it can be a view operation. It makes sense that perhaps for the packed layout you can't have strides, but this seems to be just the usual triplet?

Reply:
OK, since `transpose` is cheap, we'll just do this when we run the kernel, removed the transposed layout